### DISCUSSION TITLE
Vue Flux applicatifs: Manque les icones personnalisées  ?

### DISCUSSION THREAD

**Discussion by lynch4fr** [dbarzin/mercator#895] [General]
Bonjour
J'ai remarqué que les icônes personnalisées des applications ne sont pas utilisés dans la "Vue : Flux applicatifs". La vue utilise encore les anciennes icônes ?

Certainement un oubli car elles sont tout à fait prises en compte dans la "Vue : Application" et l'Exploration.

Merci d'avance.

**Comment by dbarzin** [comment]
Suivi dans : https://github.com/dbarzin/mercator/issues/896

**Comment by dbarzin** [comment]
Corrigé !
